ScangridData LeftScanner
ScangridData RightScanner
ScangridSteering Steering
ScangridStatus LeftScannerStatus
ScangridStatus RightScannerStatus